src/VisualStudio/Core/Impl/Options/OptionPreviewControl.xaml

```diff
@@ -26,56 +26,6 @@
             </StackPanel>
         </DataTemplate>
 
-        <DataTemplate DataType="{x:Type options:CheckBoxWithComboOptionViewModel}">
```

Comment:
so we're not using this anywhere i guess?

Reply:
Not that I could find.